src/DependencyInjection/Configuration.php

```diff
@@ -624,6 +624,18 @@ private function addSharedPluginNodes(ArrayNodeDefinition $pluginNode, $disableA
             ->end()
         ->end();
         // End error plugin
+
+        $throttle = $children->arrayNode('throttle')
+            ->canBeEnabled()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('name')->end()
```

Comment:
looking at the test failure: what is the name here? and what is key?

the name is not made required, but the configuration class does not handle a missing name field.

Reply:
@dbu You need to set name, name this is required option

Reply:
then we are missing `isRequired()` 

can you please add an `info()` as well with an explanation what this name is? the name of what thing do we specify here?

Reply:
Done, check #462